pkg/hns/netconf_windows.go

```diff
@@ -26,9 +29,15 @@ import (
 // NetConf is the CNI spec
 type NetConf struct {
 	types.NetConf
+	ApiVersion    int                  `json:"ApiVersion"`
 	HcnPolicyArgs []hcn.EndpointPolicy `json:"HcnPolicyArgs,omitempty"`
 	Policies      []policy             `json:"policies,omitempty"`
 	RuntimeConfig RuntimeConfig        `json:"runtimeConfig"`
+	OptionalFlags OptionalFlags        `json:"optionalFlags"`
```

Comment:
Is there any reason to push this down a level, configuration-wise? It seems LoopbackDSR could just be on this level?

Reply:
No good ones, I'll change it back.